packages/block-library/src/navigation/index.php

```diff
@@ -1504,19 +1490,6 @@ function block_core_navigation_mock_parsed_block( $inner_blocks, $post ) {
 function block_core_navigation_insert_hooked_blocks( $inner_blocks, $post ) {
 	$mock_navigation_block = block_core_navigation_mock_parsed_block( $inner_blocks, $post );
 
-	if ( function_exists( 'apply_block_hooks_to_content' ) ) {
```

Comment:
[`apply_block_hooks_to_content`](https://developer.wordpress.org/reference/functions/apply_block_hooks_to_content/) has been [part of WP Core since 6.6](https://github.com/WordPress/wordpress-develop/blob/4b3436968d60fc4dc301f3fe1ac888fbe6d59bce/src/wp-includes/blocks.php#L1037-L1052) -- which is now the minimum requirement for Gutenberg, so it's guaranteed to exist.